app/dturn/dturn.js

```diff
@@ -125,10 +125,11 @@ function insertStatementId(discussionId, userId, statementId) {
 
   // Only run updates if participants or round changes
   const participants = Object.keys(Discussions[discussionId].Uitems).length
-  const lastRound = Object.keys(Discussions[discussionId].ShownStatements).length - 1
+  const lastRound =
+    Discussions[discussionId].lastRound || Object.keys(Discussions[discussionId].ShownStatements).length - 1
```

Comment:
0 is a legitimate value for lastRound so how about ?? instead of || so it doesn't do Object.keys so much.